src/org/loklak/api/search/GenericScraper.java

```diff
@@ -48,6 +48,8 @@
 
 public class GenericScraper extends HttpServlet {
 
+	private static final long serialVersionUID = 4653635987712691127L;
```

Comment:
It is autogenerated and not having it gives a warning. @daminisatya hadn't added it in her PR, so I did that.